tests/Jellyfin.Naming.Tests/Video/ExtraTests.cs

```diff
@@ -45,44 +45,35 @@ public void TestExpandedExtras()
             Test("300-behindthescenes.mp4", ExtraType.BehindTheScenes, videoOptions);
         }
 
-	[Fact]
-        public void TestDirectories()
+	[Theory]
+        [InlineData(ExtraType.BehindTheScenes, "behind the scenes" )]
+        [InlineData(ExtraType.DeletedScene, "deleted scenes" )]
+        [InlineData(ExtraType.Interview, "interviews" )]
+        [InlineData(ExtraType.Scene, "scenes" )]
+        [InlineData(ExtraType.Sample, "samples" )]
+        [InlineData(ExtraType.Clip, "shorts" )]
+        [InlineData(ExtraType.Clip, "featurettes" )]
+        [InlineData(ExtraType.Unknown, "extras" )]
+        public void TestDirectories(ExtraType type, string dirName)
         {
             var videoOptions = new NamingOptions();
```

Comment:
This class is very expensive to construct, it compiles 10+ regexes. We should only construct it once per test class to reduce the amount of time it takes to execute the tests.

Reply:
Refactored the instantiation out to a readonly field in the test class.